infinispan-remote/query/src/main/java/org/infinispan/tutorial/simple/remote/query/InfinispanRemoteQuery.java

```diff
@@ -101,6 +105,31 @@ static List<Person> queryWithWhereStatementOnValues() {
       return queryResult;
    }
 
+   private static void queryWithScoreAndFilterOnNestedValues() {
+      System.out.println("== Query and filter on nested values of the team");
+      Query<Team> query = teamCache.query("FROM tutorial.Team t join t.players p where p.bornIn = :bornIn");
+      // Set the parameter value
+      query.setParameter("bornIn", "London");
+      List<Team> teamsWithPeopleFromLondon = query.execute().list();
+      System.out.println("There are only 2 teams " + teamsWithPeopleFromLondon.size());
+      System.out.println(teamsWithPeopleFromLondon);
+
+      // Get the score
+      Query<Object[]> queryWithProjectionAndScore = teamCache.query("select t.teamName, score(t) FROM tutorial.Team t where t.points=88");
+      List<Object[]> results = queryWithProjectionAndScore.execute().list();
+      System.out.println("Team with 88 points: " + results.size());
+      System.out.println("Team name: " + results.get(0)[0]);
+      System.out.println("Score: " + results.get(0)[1]);
+
+      // With filter in nested values
+      Query<Object[]> queryProjection = teamCache.query("select t.teamName, score(t) FROM tutorial.Team t join t.players p where p.lastName = :lastName");
+      queryProjection.setParameter("lastName", "Granger");
+      results = queryProjection.execute().list();
+      System.out.println("Hermione number of teams: " + results.size());
```

Comment:
@fax4ever this is not working the same in the test suite than the real server. results look the same when I try with the rest api and hotrod.